diff --git a/src/stringify.js b/src/stringify.js
index df291fdf1a10d076dae1cb2dcdb717b2c9fd042f..78af3d852e64e7f1a1e1b31062bdab6217d01cf6 100644
--- a/src/stringify.js
+++ b/src/stringify.js
@@ -1,6 +1,5 @@
 import {
 	DevalueError,
-	enumerable_symbols,
 	get_type,
 	is_plain_object,
 	is_primitive,
@@ -175,13 +174,6 @@ export function stringify(value, reducers) {
 						);
 					}
 
-					if (enumerable_symbols(thing).length > 0) {
-						throw new DevalueError(
-							`Cannot stringify POJOs with symbolic keys`,
-							keys
-						);
-					}
-
 					if (Object.getPrototypeOf(thing) === null) {
 						str = '["null"';
 						for (const key in thing) {
diff --git a/src/uneval.js b/src/uneval.js
index ce3ef34aee0ed12405c8b72d0dbee13cb02f8825..dbde0b82758e8846f40be687081aca5456b8316e 100644
--- a/src/uneval.js
+++ b/src/uneval.js
@@ -1,6 +1,5 @@
 import {
 	DevalueError,
-	enumerable_symbols,
 	escaped,
 	get_type,
 	is_plain_object,
@@ -107,13 +106,6 @@ export function uneval(value, replacer) {
 						);
 					}
 
-					if (enumerable_symbols(thing).length > 0) {
-						throw new DevalueError(
-							`Cannot stringify POJOs with symbolic keys`,
-							keys
-						);
-					}
-
 					for (const key in thing) {
 						keys.push(stringify_key(key));
 						walk(thing[key]);
