diff --git a/src/stringify.js b/src/stringify.js
index 8ce427e6f6b8573e6aa19db6671191c461e55813..f987f4e6252b832a539a1662add4941d2696d5fe 100644
--- a/src/stringify.js
+++ b/src/stringify.js
@@ -141,13 +141,6 @@ export function stringify(value, reducers) {
 						);
 					}
 
-					if (Object.getOwnPropertySymbols(thing).length > 0) {
-						throw new DevalueError(
-							`Cannot stringify POJOs with symbolic keys`,
-							keys
-						);
-					}
-
 					if (Object.getPrototypeOf(thing) === null) {
 						str = '["null"';
 						for (const key in thing) {
diff --git a/src/uneval.js b/src/uneval.js
index 21a98a44069714f3d1420b127c5427b6101feaed..a5a94ce7caf3cdcb4dd62c1939cc1f43ab0a3d2e 100644
--- a/src/uneval.js
+++ b/src/uneval.js
@@ -89,13 +89,6 @@ export function uneval(value, replacer) {
 						);
 					}
 
-					if (Object.getOwnPropertySymbols(thing).length > 0) {
-						throw new DevalueError(
-							`Cannot stringify POJOs with symbolic keys`,
-							keys
-						);
-					}
-
 					for (const key in thing) {
 						keys.push(`.${key}`);
 						walk(thing[key]);
